nxrm-ha/values.yaml

```diff
@@ -268,4 +268,4 @@ nexus:
 config:
   enabled: false
   mountPath: /sonatype-nexus-conf
-  data: []
\ No newline at end of file
+  data: {}
```

Comment:
Please add an empty new line to the end of this file

Reply:
Done